pkg/plugins/optional/grafana/alphav1/scaffolds/internal/templates/overall.go

```diff
@@ -0,0 +1,1449 @@
+          "calcs": ["lastNotNull"],
+          "fields": "",
+          "values": false
+        },
+        "showThresholdLabels": false,
+        "showThresholdMarkers": true
+      },
+      "pluginVersion": "8.4.3",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "P1809F7CD0C75ACF3"
+          },
+          "exemplar": true,
+          "expr": "workqueue_depth{job=\"$job\", namespace=\"$namespace\"}",
+          "interval": "",
+          "legendFormat": "",
+          "refId": "A"
+        }
+      ],
+      "title": "Current WorkQueue Depth",
+      "type": "gauge"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "P1809F7CD0C75ACF3"
+      },
+      "description": "How long in seconds an item stays in workqueue before being requested",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "palette-classic"
+          },
+          "custom": {
+            "axisLabel": "",
+            "axisPlacement": "auto",
+            "barAlignment": 0,
+            "drawStyle": "line",
+            "fillOpacity": 10,
+            "gradientMode": "none",
+            "hideFrom": {
+              "legend": false,
+              "tooltip": false,
+              "viz": false
+            },
+            "lineInterpolation": "linear",
+            "lineWidth": 1,
+            "pointSize": 5,
+            "scaleDistribution": {
+              "type": "linear"
+            },
+            "showPoints": "auto",
+            "spanNulls": false,
+            "stacking": {
+              "group": "A",
+              "mode": "normal"
+            },
+            "thresholdsStyle": {
+              "mode": "off"
+            }
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          },
+          "unit": "s"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 3,
+        "y": 1
+      },
+      "id": 5,
+      "options": {
+        "legend": {
+          "calcs": ["max", "mean"],
+          "displayMode": "list",
+          "placement": "right"
+        },
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "P1809F7CD0C75ACF3"
+          },
+          "exemplar": true,
+          "expr": "histogram_quantile(0.50, sum(rate(workqueue_queue_duration_seconds_bucket{job=\"$job\", namespace=\"$namespace\"}[5m])) by (instance, name, le))",
+          "interval": "",
+          "legendFormat": "P50 {{name}} {{instance}} ",
```

Comment:
An error happens when `kb init ... --plugins=...,grafana.common.kubebuilder.io/v1-alpha`:

> failed to initialize project: unable to scaffold with "grafana.common.kubebuilder.io/v1-alpha": template: *templates.OverallManifest:188: function "name" not defined

It looks like: for the curly braces here `{{...}}` in the string literal, the Go runtime will consider this as template which aims to fill in variables.

I tried the basic usage of [embed](https://pkg.go.dev/embed) which still came this error. Wondering if someone can help with this or some suggestions on handling such?

Reply:
Can you check if {{"{{output}}"}} solves the issue?

Reply:
Yeah, "{{name}}" here is merely string, which is not necessary to be filled with values.

For this example, the grafana will receive:
```json
{
  "legendFormat": "P50 {{name}} {{instance}}"
}
```

So then the grafana engine will consider this as a template to fill the corresponding values from the metrics and the query.

In my panel, it can be:
<img width="500" alt="Screen Shot 2022-06-13 at 7 20 24 PM" src="https://user-images.githubusercontent.com/18136486/173462992-4923dcb3-fd15-41f7-be0c-2071d50cedfa.png">

See here the `cronjob` the value of "name" and the `172.10.0.13:8443` is the value of "instance", they are generated by grafana.

Reply:
So this is merely an option for displaying the panel.
We can leave `legendFormat` as blank and the label will look like:
<img width="287" alt="Screen Shot 2022-06-13 at 7 23 25 PM" src="https://user-images.githubusercontent.com/18136486/173463246-2167b6d4-e63d-4485-a11b-55090364af79.png">

Reply:
> Can you check if {{"{{name}}"}} solves the issue?

This exactly solves the issue! That's awesome! Thx @varshaprasad96